canopy/design-system/src/styles/layout.scss

```diff
@@ -60,3 +60,79 @@ $sizes: (
     padding-left: $size;
   }
 }
+
+@for $i from 0 through 5 {
+  .border-#{$i} {
+    border: ($i * 1px) solid;
+  }
+
+  .borderTop-#{$i} {
+    border-top: ($i * 1px) solid;
+  }
+
+  .borderRight-#{$i} {
+    border-right: ($i * 1px) solid;
+  }
+
+  .borderBottom-#{$i} {
+    border-bottom: ($i * 1px) solid;
+  }
+
+  .borderLeft-#{$i} {
+    border-left: ($i * 1px) solid;
+  }
+
+  .borderWidth-#{$i} {
+    border-width: $i * 1px;
+  }
+
+  .borderTopWidth-#{$i} {
+    border-top-width: $i * 1px;
+  }
+
+  .borderRightWidth-#{$i} {
+    border-right-width: $i * 1px;
+  }
+
+  .borderBottomWidth-#{$i} {
+    border-bottom-width: $i * 1px;
+  }
+
+  .borderLeftWidth-#{$i} {
+    border-left-width: $i * 1px;
+  }
+}
+
+.borderStyle-solid {
+  border-style: solid;
+}
+
+.borderColor-smoke {
+  border-color: $smoke;
+}
+
+@for $i from 0 through 100 {
```

Comment:
I don't see how these would be particularly useful. The design system shouldn't strive to turn every possible CSS declaration into a class.